modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/parcellationmeshgeneration/v0.1.0.json

```diff
@@ -0,0 +1,51 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/atlas/parcellationmeshgeneration/v0.1.0/shapes/"
+    }
+  ],
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0"
```

Comment:
The latest version of [neurosciencegraph/commons/activity](https://github.com/INCF/neuroshapes/blob/master/modules/commons/src/main/resources/schemas/neurosciencegraph/commons/activity/v0.1.4.json) should be used.